### DISCUSSION TITLE
304 Not Modified with REST

### DISCUSSION THREAD

**Discussion by Miguelklappes** [directus/directus#5167] [Archive]
I am on the hosted version of Directus, I keep getting a 304 Not  Modified from the API everytime I try to get data from  it, nothing changes, I deleted the collection and it's still there.
---

### Project details
Directus Version: 9.0.0-rc.59
Environment: production
OS: Linux 4.4.0
Node: 15.14.0

**Comment by benhaynes** [comment]
When you say you're on the "Hosted Version", do you mean Directus Cloud?

**Comment by rijkvanzanten** [comment]
Directus Cloud has caching enabled by default. You can skip the cache by using the `Cache-Control: no-cache` header 👍🏻
